.github/workflows/odh-notebooks-sync.yml

```diff
@@ -92,7 +91,28 @@ jobs:
             for dir in "${directories[@]}"; do
               counter=$((counter+1))
               echo "--Processing directory $counter '$dir' of total $total"
-              cd "$dir" && pipenv install ${package_name}~="${CODEFLARE_RELEASE_VERSION}" && pipenv --rm && cd -
+              cd "$dir"
+              pipfile_python_version=$(grep -E '^python_version' ./Pipfile | cut -d '"' -f 2)
+              if [[ "$pipfile_python_version" > 3.8 ]]; then
```

Comment:
Thinking that this might actually break in the future if we upgrade to python 3.10. The comparison made here will work as:
- 3.9 is greater than 3.8
- 3.10 is not greater than 3.8

Reply:
Thank you @ChristianZaccaria for pointing this out, I have added some more if statements to consider this usecase.. Please review!

Reply:
Tested 3.10 usecase locally, the pipenv installation worked seamlessly for 3.10 as well!
![image](https://github.com/project-codeflare/codeflare-sdk/assets/84722973/94414cfc-683b-4407-95f1-21c97e000da9)